frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/clearingStatus.jspf

```diff
@@ -7,13 +6,14 @@
   ~ which accompanies this distribution, and is available at
   ~ http://www.eclipse.org/legal/epl-v10.html
 --%>
+<%@ page import="org.eclipse.sw360.portal.common.PortalConstants" %>
 <portlet:resourceURL var="deleteReleaseAjaxURL">
     <portlet:param name="<%=PortalConstants.ACTION%>" value='<%=PortalConstants.DELETE_RELEASE%>'/>
 </portlet:resourceURL>
 
 <core_rt:set var="clearingTeamsStringSet" value='<%=PortalConstants.SET_CLEARING_TEAMS_STRING%>'/>
 
-<jsp:useBean id="releasesAndProjects" type="java.util.Map<org.eclipse.sw360.datahandler.thrift.components.Release,java.lang.String>" scope="request"/>
+<jsp:useBean id="releasesAndProjects" type="java.util.Map<org.eclipse.sw360.datahandler.thrift.components.Release,java.lang.String[]>" scope="request"/>
```

Comment:
That's a quick and dirty solution. You're absolutely right: I should've cleaned that up before creating PR